admin_guide/encrypting_data.adoc

```diff
@@ -159,58 +158,58 @@ resources:
     - identity: {}
 ----
 
-To create a new secret perform the following steps:
+To create a new secret:
 
-. Generate a 32 byte random key and base64 encode it. For example, on Linux and
-macOS the following command can be used:
+. Generate a 32-byte random key and base64 encode it. For example, on Linux and
+macOS use:
 +
 ----
-head -c 32 /dev/urandom | base64
+$ head -c 32 /dev/urandom | base64
 ----
 +
 [IMPORTANT]
 ====
-The encryption key must be generated with an appropriate cryptographically secure random number
-generator like *_/dev/urandom_*. For example, `math/random` from Golang or `random.random()` from
-Python are not suitable.
+The encryption key must be generated with an appropriate cryptographically
+secure random number generator like *_/dev/urandom_*. For example, `math/random`
+from Golang or `random.random()` from Python are not suitable.
 ====
 
-. Place that value in the *secret* field
+. Place that value in the `secret` field.
 
-. Restart API server
```

Comment:
@php-coder What must be run to restart the API server?

Reply:
Based on our documentation, looks like the commands are different and depend on the setup:
1) for OpenShift Origin the command is `systemctl restart origin-master`
2) for OpenShift Enterprise on a single master cluster installation the command is `systemctl restart atomic-openshift-master`
3) for OpenShift Enterprise on a multi-master cluster installation the command will be `systemctl restart atomic-openshift-master-api` (on each master)

Here is the possible asciidoc variant:
```
ifdef::openshift-origin[]
----
# systemctl restart origin-master
----
endif::[]
ifdef::openshift-enterprise[]
On a single master cluster installation:
----
# systemctl restart atomic-openshift-master
----
On a multi-master cluster installation, on each master:
----
# systemctl restart atomic-openshift-master-api
----
endif::[]
```

@smarterclayton Could you confirm that these commands are correct?